beacon-chain/rpc/validator/attester.go

```diff
@@ -211,19 +217,21 @@ func (vs *Server) SubscribeCommitteeSubnets(ctx context.Context, req *ethpb.Comm
 		return uint64(len(vals)), nil
 	}
 
-	// run first request
+	// Request the head validator indices of epoch represented by the first requested
+	// slot.
 	currValsLen, err := fetchValsLen(req.Slots[0])
 	if err != nil {
-		return nil, err
+		return nil, status.Errorf(codes.Internal, "Could not retrieve head validator indices: %v", err)
```

Comment:
```suggestion
		return nil, status.Errorf(codes.Internal, "Could not retrieve head validator length: %v", err)
```

`fetchValsLen` fetches the head validator length